tsconfig.build.cjs.json

```diff
@@ -1,6 +1,7 @@
 {
   "compilerOptions": {
-    "composite": true
+    "composite": true,
+    "verbatimModuleSyntax": false,
```

Comment:
does tsconfig.json support comments (I know it supports trailing commas so it's not pure JSON)? Might be worth calling this one out.